CodeGeneration/Sources/SyntaxSupport/TokenSpec.swift

```diff
@@ -21,16 +21,48 @@ public struct TokenSpec {
     case other
   }
 
+  /// The name of the token, suitable for use in variable or enum case names.
   public let varOrCaseName: TokenSyntax
 
-  /// If `true`, this is for an experimental language feature, and any public
-  /// API generated should be SPI.
+  /// Indicates if the token is part of an experimental language feature.
+  ///
+  /// If `true`, this keyword is for an experimental language feature, and any public
+  /// API generated should be marked as SPI
   public let isExperimental: Bool
 
+  /// The name of the token that can be shown in diagnostics.
   public let nameForDiagnostics: String
+
+  /// The actual text of the token, if available.
   public let text: String?
+
+  /// The kind of the token.
   public let kind: Kind
 
+  /// The text of the token enclosed in double quotes, if available.
+  ///
+  /// This property is used in code generation where a string representation of the token is needed.
+  public var quotedText: TokenSyntax? {
+    guard let text = text else { return nil }
+    return "\"\(raw: text)\""
+  }
```

Comment:
Same comment as in https://github.com/apple/swift-syntax/pull/2076#discussion_r1296182022

Reply:
Great point. Done